Cargo.toml

```diff
@@ -25,6 +25,7 @@ lazy_static = "1.3"
 paste = "0.1"
 regex = "1.1"
 serde = { version = "1.0", features = ["derive", "rc"], optional = true }
+nom = "5.0.1"
```

Comment:
Probably don't want to pin a specific version like this, assuming they follow semver that is.